<launch>    
  <arg name="odom_frame_id"   default="odom"/>
  <arg name="base_frame_id"   default="base_footprint"/>
  <arg name="global_frame_id" default="map"/>
  <arg name="odom_topic" default="odom" />
  <arg name="laser_topic" default="mybot/laser/scan" />

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find lan_nav)/config/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find lan_nav)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find lan_nav)/config/local_costmap_params.yaml"  command="load" />
    <rosparam file="$(find lan_nav)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find lan_nav)/config/base_local_planner_params.yaml" command="load" />
    
    <remap from="cmd_vel" to="/mobile_base/commands/velocity"/>	
    <remap from="map" to="/map" />
    <remap from="odom" to="$(arg odom_topic)"/>
    <remap from="scan" to="$(arg laser_topic)"/>
  </node>

</launch>
